README.md

```diff
@@ -75,332 +75,336 @@ helm uninstall my-release
+| `expose.loadBalancer.ports.httpsPort`                                | The service port Harbor listens on when serving HTTPS                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | `30002`                               |
+| `expose.loadBalancer.annotations`                                    | The annotations attached to the loadBalancer service                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | {}                                    |
+| `expose.loadBalancer.sourceRanges`                                   | List of IP address ranges to assign to loadBalancerSourceRanges                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | []                                    |
+| `expose.loadBalancer.annotations`                                    | The annotations used commonly for loadBalancer                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                       |
```

Comment:
line 108 is the same config of line 106? one should be `expose.loadBalancer.annotations`, and the other should be `expose.loadBalancer.labels`, I suppose.